modules/platform/nuxeo-platform-usermanager-api/src/main/java/org/nuxeo/ecm/platform/usermanager/NuxeoPrincipalImpl.java

```diff
@@ -493,4 +505,9 @@ private Object writeReplace() throws ObjectStreamException {
             return new DataTransferObject(this);
         }
     }
+
+    @Override
+    public boolean isPartial() {
+        return isPartial;
+    }
```

Comment:
Could you move it before the `TransferableClone`? Also, could you move its setter close to it?